docs/source/platform/schema-validation.md

```diff
@@ -131,15 +131,37 @@ Of course, it’s also possible to leave the field in place indefinitely!
 
 <h2 id="cli">Checking schema changes with the Apollo CLI</h2>
 
-To check and see the difference between the current published schema and a new version, run the following command, substituting the appropriate GraphQL endpoint URL and an API key:
+To check and see the difference between the current published schema and a new version, make sure you have a valid [service configuration](../resources/apollo-config.html#service-config) like so, substituting the appropriate GraphQL endpoint URL and an API key:
+
+```bash
+# .env
+ENGINE_API_KEY=<API_KEY>
```

Comment:
I think we need to be more explicit that this line goes inside of an `.env` file and where the `.env` file should be saved. Additionally it can be confusing what the api key is, something like [this](https://github.com/apollographql/apollo/pull/291/files#diff-c3154587c474c91ac2bac208f0ed3accL20) helps a lot